src/app/vginventory/src/hooks/useCreateVideoGame.ts

```diff
@@ -36,10 +38,12 @@ export function useCreateVideoGame() {
     }
 
     const updateDescription = (event:any) => {
+        event.preventDefault();
         setDescription(event.target.value);
     }
 
     const updateReleaseDate = (event:any) => {
+        event.preventDefault();
```

Comment:
This causes "event.preventDefault is not a function" exception because prop is not an event type.

Reply:
I forgot to update this function when I updated the React Component type. The onChange event does not return an event instead it returns a moment obj if valid date, if not a string